e2e/cypress/integration/enterprise/cloud/onboarding/enter_support_email_spec.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @onboarding @cloud_only
+
+describe('Cloud Onboarding - Sysadmin enter support email', () => {
+    let townSquarePage;
+    let sysadmin;
+
+    before(() => {
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        cy.apiInitSetup().then(({team}) => {
+            townSquarePage = `/${team.name}/channels/town-square`;
+        });
+
+        cy.apiAdminLogin().then((res) => {
+            sysadmin = res.user;
+        });
+    });
+
+    beforeEach(() => {
+        // # Login as sysadmin and set all steps to false
+        const preference = {
+            user_id: sysadmin.id,
+            category: 'recommended_next_steps',
+            value: 'false',
+        };
+        const adminSteps = [
+            'complete_profile',
+            'notification_setup',
+            'team_setup',
+            'invite_members',
+            'enter_support_email',
+            'hide',
+            'skip',
+        ];
+
+        cy.apiSaveUserPreference(adminSteps.map((step) => ({...preference, name: step})));
+        cy.visit(townSquarePage);
+    });
+
+    it('type invalid email address', () => {
```

Comment:
Should we add a happy case that the support email was properly updated?

Reply:
I have added one